pkg/install/install.go

```diff
@@ -258,6 +262,16 @@ func Install(ctx context.Context, image string, opts *Options) error {
 	return nil
 }
 
+func validateServiceLBAnnotations(annotations []string) error {
+	for _, annotation := range annotations {
+		split := strings.Split(annotation, "=")
```

Comment:
Is it valid for the value of an annotation to have an `=`? If so, I think `strings.Cut` here makes sense instead.

Reply:
Yep! Its valid. Great catch.